templates/travis/.travis/release.sh.j2

```diff
@@ -0,0 +1,58 @@
+if [ $# -eq 0 ]; then
+    echo "No arguments provided"
+    exit 1
+fi
+
+export PLUGIN_DIR={{ plugin_name }}
+export BASEDIR=$(dirname $(readlink -f "$0"))
+export VERSION=$1
+
+cd $BASEDIR/..
+
+towncrier --version $VERSION --draft
+
+while true; do
+    read -p "Does the changelog draft looks good? [Y/N] " yn
+    case $yn in
+        [Yy]* ) break;;
+        [Nn]* ) exit;;
+        * ) echo "Please answer yes or no.";;
+    esac
+done
+
+git checkout -b "${VERSION}_release"
+
+# First commit: changelog
+towncrier --yes --version $VERSION
+git add .
+git commit -F- <<EOF
+Building changelog for $VERSION
+
+[noissue]
+EOF
+
+# Second commit: release version
+sed -i "s/\.dev//g" setup.py
+# Missing sed on pulpcore requirement on setup.py
```

Comment:
I could not think of a good sed for this

Reply:
maybe something like
`sed -i -e '/version=/s/\.dev//'
To only perform it once and on a line that contains "version=".

Reply:
Sorry, I did not describe well the problem, what I need to do is find a way to:
`pulpcore>=3.3` into this `pulpcore>=3.4,<3.5`